tigerpath/majors_and_certificates/majors/NEU_2018.json

```
@@ -0,0 +1,348 @@
+            "explanation": "To enter the neuroscience concentration, students must have completed NEU 201 Fundamentals of Neuroscience, NEU 202 Introduction to Cognitive Neuroscience, and MAT 103 Calculus I.",
+            "req_list": [
+                {
+                    "name": null,
+                    "max_counted": 1,
+                    "min_needed": "ALL",
+                    "explanation": "To enter the neuroscience concentration, students must have completed NEU 201 Fundamentals of Neuroscience, NEU 202 Introduction to Cognitive Neuroscience, and MAT 103 Calculus I.",
+                    "course_list": [
+                        "NEU 202",
+                        "MAT 103"
+                    ]
+                },
+                {
+                    "name": null,
+                    "max_counted": 1,
+                    "min_needed": "ALL",
+                    "explanation": "ISC 236 - Biochemistry and Neuroscience offers an alternative to NEU 201 (formerly NEU 258) (a prerequisite).",
+                    "course_list": [
+                        "NEU 201",
+                        "ISC 236"
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "Department Requirements",
+            "max_counted": 1,
+            "min_needed": "ALL",
+            "explantion": "",
+            "req_list": [
+                {
+                    "name": "Laboratory in Principles of Neuroscience",
+                    "max_counted": 1,
+                    "min_needed": "ALL",
+                    "explanation": "",
+                    "completed_by_semester": 8,
+                    "course_list": [
+                        "NEU 350"
+                    ]
+                },
+                {
+                    "name": "Mathematical Tools for Neuroscience",
+                    "max_counted": 1,
+                    "min_needed": "ALL",
+                    "explanation": "",
+                    "completed_by_semester": 8,
+                    "course_list": [
+                        "NEU 314"
+                    ]
+                },
+                {
+                    "name": "Physics",
+                    "max_counted": 1,
+                    "min_needed": "ALL",
+                    "explanation": "",
+                    "completed_by_semester": 8,
+                    "req_list": [
+                        {
+                            "name": null,
+                            "max_counted": 1,
+                            "min_needed": "ALL",
+                            "explanation": "",
+                            "completed_by_semester": 8,
+                            "course_list": [
+                                "PHY 101",
+                                "PHY 102"
+                            ]
+                        },
+                        {
+                            "name": null,
+                            "max_counted": 1,

```

If all of the ISC courses are satisfied, they replace two courses: PHY101 and PHY102, right?
```suggestion
                            "max_counted": 2,
```